packages/cacti-copm-test/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@hyperledger-cacti/cacti-copm-test",
```

Comment:
I believe this should also be in examples. Considering this is only testing by calling client commands.

Reply:
The PR follows the example of `cactus-plugin-ledger-connector-corda`, which is tested by `cactus-test-tooling' in the packages directory.  Other packages, such as the besu connector, have their own test directory under packages.  Please correct me if I'm wrong, @petermetz.

Reply:
Oh okay, if you have discussed with @petermetz and he is fine with it then I'm also fine.